app/src/main/java/org/mozilla/focus/media/MediaSessionService.kt

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.media
+
+import mozilla.components.browser.state.store.BrowserStore
+import mozilla.components.feature.media.service.AbstractMediaSessionService
+import org.mozilla.focus.ext.components
+
+/**
+ * [AbstractMediaSessionService] implementation for injecting [BrowserStore] singleton.
+ */
+class MediaSessionService : AbstractMediaSessionService() {
```

Comment:
You need to register the service in the AndroidManifest too. (example [here](https://github.com/mozilla-mobile/reference-browser/blob/master/app/src/main/AndroidManifest.xml#L177-L178))